components/pageTitle.js

```diff
@@ -0,0 +1,30 @@
+import React, {useState} from 'react';
+import {StyleSheet, Text, View} from 'react-native';
+import {Colors, GlobalStyles} from '../styles';
+
+export default function PageTitle({title}) {
+  return (
+    <View style={styles.content}>
+      <Text style={styles.title}>{title}</Text>
+    </View>
+  );
+}
+const styles = StyleSheet.create({
+  content: {
+    ...GlobalStyles.boxShadow,
+    ...GlobalStyles.content,
```

Comment:
Similar to a prior comment, I'd take out the GlobalStyles.content from the component itself. I think it's easier to use/compose components when they don't implicitly assume any relationship (i.e. margins, etc) to other components in their design. Instead, let the parent component define it.